### ISSUE TITLE
Form validation: legend float-left with radio buttons

### ISSUE THREAD

**Issue by shawnthompson** [wet-boew/wet-boew#7975]
I found a new issue (reason) not to have the `legend` `float: left`.

If you look on the form validation page at the checkbox and radio sample, when you add a `legend` directly above a `checkbox` or `radio` class you can't easily select the text inside the `legend`, let alone add a link to it.

We can either turn the float off the `legend `or remove the position from the `.checkbox` and [.radio](url) but we would have to look at why they are added there in Bootstrap.

```
.checkbox, .radio {
  display: block;
  margin-bottom: 10px;
  margin-top: 10px;
  position: relative;
}
```
Discussed in #7766 why it's added.

CC: @kodecount @duboisp

**Comment by kodecount** [comment]
Yeah we determined that changing this affected the original style. I forget how it did that.

**Comment by shawnthompson** [comment]
@kodecount Having this stops us from using `fieldset` inside `fieldset`. You talked about a bug in firefox and `fieldset`. 

Do you know which one? There seems to be a lot of bugs with `fieldset` and firefox.

https://bugzilla.mozilla.org/buglist.cgi?quicksearch=fieldset

**Comment by kodecount** [comment]
Sorry I don't remember, I've been away from the project since my contract was over with the government.

**Comment by shawnthompson** [comment]
Any update on this? Someone inquired about it, saying they were having a difficult time select the text of the first `label` after a `legend`

**Comment by NoahGresser** [comment]
> Any update on this? Someone inquired about it, saying they were having a difficult time select the text of the first `label` after a `legend`

Slight correction Shawn, it was actually that the `legend` text could not be selected, but we determined that the reason it could not be selected was for some reason the use of `float: left` on the `legend` made it so that the first `div` element with `class="checkbox"` or `class="radio"` immediately after that `legend` would have a height (one not directly defined by any CSS rule) that would make its main area box (not the margin, border or padding) overlap and cover the `legend` text, even though the actual `label` element inside that `div` had a normal height.
